feature/favourites/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,2 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest package="com.epmedu.animeal.favourites" />
```

Comment:
Please use namespace in build.gradle instead of declaring package in manifest. You can find an example in other modules

Reply:
removed from manifest, used in build.gradle